### PR TITLE
[Create a subscription] - Tab Meeting Recording and Transcript (nodeJs) - Final

### PR DESCRIPTION
**Pull request by Jegadeesh-MSFT**
…s) - Final

### CODE REVIEW

**Comment by Jegadeesh-MSFT** [comment]
image
**Executing build pipepline**
https://github.com/OfficeDev/Microsoft-Teams-Samples/actions/runs/6954108656

File: samples/tab-meeting-transcript-recording/nodejs/src/components/meetingtranscriptrecording.js [hunk 1]
```diff
@@ -79,11 +139,71 @@ const MeetingTranscriptRecording = () => {
...
+                .then(data => {
+                        if (data.error == "consent_required") {
+                            setIsConsentButtonVisible(true);
+                            setIsLoginVisible(false);
+                            setLoading(false);
+                        }
+                        if (data) {
+                            return;
+                        } else {
+                            return;
```
Comment (Pawank-MSFT) [inline samples/tab-meeting-transcript-recording/nodejs/src/components/meetingtranscriptrecording.js:206]: Why we are using if else condition, not meaning of this code.

**Review by Pawank-MSFT — APPROVED** [review]
please fix this small comments, Approving with this comment

File: samples/tab-meeting-transcript-recording/nodejs/src/components/meetingtranscriptrecording.js (hunk 1 above)
Comment (Jegadeesh-MSFT) [reply samples/tab-meeting-transcript-recording/nodejs/src/components/meetingtranscriptrecording.js:203]: Yes , that If part can be avoided. Updated it

**Comment by Pawank-MSFT** [comment]
Executed build Pipeline
https://github.com/OfficeDev/Microsoft-Teams-Samples/actions/runs/7027273548

**Pull request merged**